### PR TITLE
Update scheduler.js to use location using later versions of serverless-offline

### PR DESCRIPTION
**Pull request by david-dacostad**
Constructor Name has changed... it should be... "ServerlessOffline"

### CODE REVIEW

File: lib/scheduler.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ class Scheduler {
 
   run() {
     const offlinePlugin = this.serverless.pluginManager.getPlugins()
-      .find((p) => p.constructor && p.constructor.name === "Offline");
+      .find((p) => p.constructor && p.constructor.name === "ServerlessOffline");
```
Comment (ajmath) [inline lib/scheduler.js]: Please update to support both versions of the constructor name

Comment (david-dacostad) [reply lib/scheduler.js]: ok done...

**Comment by ajmath** [comment]
Thanks for the PR

**Comment by david-dacostad** [comment]
You are welcome...

**Pull request merged**